ToyBox/Classes/Models/Settings.cs

```diff
@@ -24,9 +24,12 @@ public class PerSaveSettings : EntityPart {
         //      Dictionary<CharacterName, 
         //          Dictionary<ClassID, HashSet<ArchetypeIDs>
 
+        // Dictionary<Character Hashcode, Dictionary<Dictionary<bodyPartName, universalParameter>, Dictionary<bodyPartName, sizeMultiplier>>
+        [JsonProperty]
+        public Dictionary<string, Dictionary<string, float>> characterSkeletonReplacers = new();
         // This is the scaling modifier which is applied to the visual model of each character
         [JsonProperty]
-        public Dictionary<string, float> characterModelSizeMultiplier = new();
+        public Dictionary<string, Tuple<float, float, float>> characterModelSizeMultiplier = new();
```

Comment:
Tuple<float, float, float> is a reference type, i.e. it's Heap Allocated.
In most cases, I prefer to use ValueTuple, which is a struct and is Stack Allocated. So basically

```cs
public Dictionary<string, (float, float, float)> characterModelSizeMultiplier = new();
```

It causes less GC pressure and has better Syntax support (and struct benefits like shallow equality)